Support cloud regions with multiple clusters in the docs (#3693)
influxdata/docs-v2: content/influxdb/cloud/reference/regions.md

<a href="https://www.influxdata.com/influxdb-cloud-2-0-provider-region/" target="_blank" class="btn">Request a cloud region</a>

{{% note %}}
#### Regions with multiple clusters

> Some InfluxDB Cloud regions have multiple InfluxDB clusters, each with a unique URL.

Reviewer: ```suggestion
Some InfluxDB Cloud regions have multiple Cloud clusters, each with a unique URL.
```

> To find out which cluster you're using, [login to your InfluxDB Cloud organization](https://cloud2.influxdata.com)

Reviewer: ```suggestion
To find your cluster URL, [log in to your InfluxDB Cloud organization](https://cloud2.influxdata.com)
```

> and look at the URL of your organization. The first subdomain identifies your 

Reviewer: ```suggestion
and review your organization URL. The first subdomain identifies your 
```

InfluxDB Cloud cluster. For example:

<pre class="highlight">

> https://<span class="bp" style="font-weight:bold;margin:0 .15rem">us-west-2-1</span>.aws.cloud2.influxdata.com/orgs/03a2bbf46249a000/...

Author: That's something we could possibly implement. I've only done this in this and one other spot, so I've kept it a manual process. Do you have other use cases for it?
